data/hai/hai missions.txt

```diff
@@ -209,6 +209,69 @@ mission "Ask the Hai about the Unfettered"
+			`	With a wave of his hand he indicates the spaceport around you. "You might have potentially noticed some difficulty landing. You see because there are lots of people who don't like outsiders putting their noses in where they're not wanted, if you happen to shoot back and accidentally destroy a Sympathizer's ship, the authorities don't much care who shot first. There are enough of those opinions to ensure that if you shoot down a Sympathizer, it is treated no differently than if you shot down any other Hai merchant. The security forces will treat it the same.`
+			`	"Now the merchants, we're a different matter. It is a topic of great and ongoing debate among us, so you might notice that some people are less offended when you shoot down someone they've been arguing with, but everyone has friends on all sides, so you should watch yourself nonetheless. Sympathizers may become upset enough to shoot at you anywhere they see you. Despite that, the simple fact of the matter is that, if you retaliate against a Hai, any Hai, it'll upset someone and count against you. Even if that Hai was a Sympathizer, and even if they shot first..."`
+			`	He looks at you very seriously to make sure what he's said has sunk in.`
+			choice
+				`	"I understand, I've stepped on some toes."`
+					goto trouble
+					to display
+						"reputation: Hai" < 1
+				`	"I understand, thank you for warning me."`
+					goto end
+					to display
+						"reputation: Hai" >= 1
+				`	"Why should I get in trouble for defending myself against Sympathizers?"`
+			`	He shrugs. "You should try not to impart human ethical standards onto Hai. The simple fact of the matter is that they are citizens, and you are not.`
+			`	"When a report is made that you have destroyed the vessel of a Hai citizen, it is up to the local security staff to verify the chain of events. If they report it was your fault, then it was your fault, unless you can challenge that in a court. However, you are not a citizen, so the security apparatus can simply brand you as a pirate and prevent you from ever gaining access to a court to challenge it in the first place. Even if you managed to do so, you're a captain; your business is in space. You would be grounded for months to work through the court proceedings to challenge it.`
+			`	"Whether you believe it is right or not, it is simply the reality of the situation. You must manage your guns and your reputation carefully where interactions with the Unfettered are concerned."`
+			
+			label trouble
+			branch "less trouble"
+				"reputation: Hai" >= 1
+				"reputation: Hai (Wormhole Access)" >= 1
+				"reputation: Hai Merchant" >= 1
+				"reputation: Hai Merchant (Human)" >= 1
+			# At least one other group besides the Sympathizers are upset.
+			`	"You have gotten yourself in trouble Captain."`
```

Comment:
I pointed it out because it might be strange to read:
`"Whether you believe it is right or not, it is simply the reality of the situation. You must manage your guns and your reputation carefully where interactions with the Unfettered are concerned."`

`"You have gotten yourself in trouble Captain."`

`He slides a data stick across to you. "Fortunately for you, in this case, I happened to see the event, and have argued that you did not know what you were getting into. This stick contains the security report file which indicates that you have been given a warning... keep this for your records, Hai Security certainly will. There will not be future warnings."`

I would remove this line.
```suggestion
```

Reply:
Hmm... We can't just swap the order of the first two paragraphs, because of the other substitutions and things, right?

```"You have gotten yourself in trouble Captain."

"Whether you believe it is right or not, it is simply the reality of the situation. You must manage your guns and your reputation carefully where interactions with the Unfettered are concerned."

He slides a data stick across to you. "Fortunately for you, in this case, I happened to see the event, and have argued that you did not know what you were getting into. This stick contains the security report file which indicates that you have been given a warning... keep this for your records, Hai Security certainly will. There will not be future warnings."```

Because if we could, that reads even better to me than removing the line.

Reply:
Oh I see what I've overlooked. Hangon, I'll make it a new suggestion.